google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -129,24 +146,41 @@ def _send_email(self, summary_data, details_data):
 
         gsuite_dwd_status = self._get_gsuite_dwd_status(summary_data)
 
-        email_content = email_util.render_from_template(
+        email_content = email_connector.render_from_template(
             'inventory_summary.jinja',
             {'inventory_index_id': self.inventory_index_id,
              'timestamp': timestamp,
              'gsuite_dwd_status': gsuite_dwd_status,
              'summary_data': summary_data,
              'details_data': details_data})
 
-        try:
-            email_util.send(
-                email_sender=email_summary_config.get('sender'),
-                email_recipient=email_summary_config.get('recipient'),
-                email_subject=email_subject,
-                email_content=email_content,
-                content_type='text/html')
-            LOGGER.debug('Inventory summary sent successfully by email.')
-        except util_errors.EmailSendError:
-            LOGGER.exception('Unable to send inventory summary email')
+        if self.notifier_config.get('email_connector'):
+            try:
+                email_connector.send(
+                    email_sender=self.notifier_config
```

Comment:
Make this call `email_connector.send()` once, as mentioned above.

Reply:
Done